python/tk_multi_publish2/api/plugins/setting.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2017 Shotgun Software Inc.
```

Comment:
New version of this: https://github.com/shotgunsoftware/tk-multi-publish2/blob/master/python/tk_multi_publish2/processing/setting.py#L16